### ISSUE TITLE
macos: don't start containers for non connected ios devices

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/mcloud-agent#277]
realted #266 for Linux was closed as won't fix because we don't have access to the usbmuxd on Linux.
But for MacOS it is not applicable. So let's try to test the presence of iOS device as part of zebrunner-farm macos shell script before starting containers.

**Comment by dhreben** [comment]
Verified.
